### ISSUE TITLE
Gave me a error while installing PearOS

### ISSUE THREAD

**Issue by omxpro** [unetbootin/unetbootin#331]
![20220116_120636](https://user-images.githubusercontent.com/69719569/149650486-fe813945-f2ea-4467-ae91-dc4133d0313d.jpg)
Is there a way to fix this?

**Comment by akovacs** [comment]
If you have access to another Windows computer, then create a USB drive using:
https://support.microsoft.com/en-us/windows/create-installation-media-for-windows-99a58364-8c02-206f-aa6f-40c3b507420d

Boot to the USB drive, and repair the Windows installation.

**Comment by Santhosh-004** [comment]
This happens only when you select unetbootin from the boot options. When you choose Windows, it continues to boot perfectly fine.

**Comment by snowythebudg** [comment]
yeah i have the same issue as well and my laptop doesnt even bother trying to even boot from usb
